services/user-sessions/src/user_sessions/serve.clj

```diff
@@ -1,63 +0,0 @@
-(ns user-sessions.serve
-  (:use [kameleon.user-session-queries]
```

Comment:
I disagree. I think putting off cleaning up obsolete code in another commit or PR makes it more likely that cruft will get left behind.
I also think it makes an unnecessary logical disconnect. Looking at a later commit or PR that just removes some obsolete code might not tell you why it became obsolete (was it replaced with something better? Or was it only used in one place and that code was removed for some reason?). If the obsolete code was removed along with the commit or PR that made it obsolete, then it's more obvious why it was removed and requires less explanation.